macros/src/attributes/global_logger.rs

```diff
@@ -43,6 +43,12 @@ fn codegen(strukt: &ItemStruct) -> TokenStream {
             <#ident as defmt::Logger>::acquire()
         }
 
+        #[no_mangle]
+        // TODO: should this be unsafe?
```

Comment:
I don't think it matters because this function won't be called directly and when it's called via FFI it will always be unsafe to call.

Reply:
So I'll go with the current convention ans will also make it unsafe